packages/vite/src/node/utils.ts

```diff
@@ -1070,27 +1070,36 @@ function backwardCompatibleWorkerPlugins(plugins: any) {
   return []
 }
 
-function deepClone<T>(value: T): T {
+type DeepWriteable<T> =
```

Comment:
Sorry to be the grammar police, but 😅 
```suggestion
type DeepWritable<T> =
```

Reply:
I first wrote writable and felt unnatural and changed to writeable😅 I should've looked up.